### PR TITLE
Anchor Arms

### PR DESCRIPTION
**Pull request by Shadowmech88**
I really always liked the buff spessman sprites, but they were always locked behind adminbus.
Now every spessman can be buff.

anchor_arms

I haven't yet found a good way to get the arms' sprite to cover jackets without also covering anything held in the hand. So if you don't want them to look weird, don't wear a jacket with them.

### CODE REVIEW

**Comment by CptWad** [comment]
oh god my sides 10/10 its too late I should go to bed im going to have a heart attack
:+1:

File: code/game/objects/items/latexballoon.dm [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 	throwforce = 0
 	w_class = 1.0
 	throw_speed = 1
 	throw_range = 15
-	var/state
 	var/datum/gas_mixture/air_contents = null
 
 /obj/item/latexballon/proc/blow(obj/item/weapon/tank/tank)
-	if (icon_state == "latexballon_bursted")
+	if (icon_state == "latexballoon_bursted")
```
Comment (Exxion) [inline code/game/objects/items/latexballoon.dm]: While you're here, please remove this icon_state check
Use a variable

File: code/game/objects/items/latexballoon.dm [hunk 2]
```diff
@@ -44,5 +45,44 @@
...
+/obj/item/latexballon/pair/attackby(obj/item/W as obj, mob/user as mob)
+	..()
+	if(istype(W, /obj/item/toy/crayon/red))
+		to_chat(user, "You color \the [src] light red using \the [W].")
+		if(src.loc == user)
+			user.drop_item(src, force_drop = 1)
+			var/obj/item/clothing/gloves/anchor_arms/A = new (get_turf(user))
+			user.put_in_hands(A)
+		else
+			new /obj/item/clothing/gloves/anchor_arms(get_turf(src.loc))
```
Comment (Exxion) [inline code/game/objects/items/latexballoon.dm:89]: > get_turf(src.loc)

I mean I guess it works but it's weird

**Comment by Exxion** [comment]
What do they look like on vox?

**Comment by Shadowmech88** [comment]
I now realize that vox require separate sprites for everything.
I'll just make vox unable to equip them.

**Comment by Exxion** [comment]
An acceptable, if boring, solution
Swole vox sprites when?

**Comment by Shadowmech88** [comment]
Actually there already are swole vox sprites.
I guess I could add vox-specific anchor arm sprites. But it's not like you'd be able to see them on most vox players; wearing a hardsuit causes the gloves layer to become invisible.

**Comment by Exxion** [comment]
Oh yeah
I forgot about that
But if there are swole vox sprites why not go ahead and make vox anchor arm sprites?

**Comment by Shadowmech88** [comment]
Honestly the arms of the swole vox aren't all that different. It'll probably amount to moving the human anchor arms down a few pixels and adding claws on the end.

**Comment by Exxion** [comment]
Works for me, as long as it lines up properly

**Comment by ihadtoregisterforthis** [comment]
What about the purple plasmamen, or the wimp greys?

**Comment by CptWad** [comment]
large pink arms on the humans look ridiculous on its own, the vox and plasmen need no changes

**Comment by Shadowmech88** [comment]
I added a vox sprite for the arms.

anchor_arms

**Comment by ihadtoregisterforthis** [comment]
:muscle::bird:

**Comment by CptWad** [comment]
and all was well in the world

**Comment by Exxion** [comment]
HAPPENING

**Comment by ihadtoregisterforthis** [comment]
And all was SWELL with the world.

**Pull request merged**